app/ui-angular/pom.xml

```
@@ -93,6 +93,7 @@
         <executions>
           <execution>
             <id>install node and npm</id>
+            <phase>initialize</phase>
```

```suggestion
            <phase>pre-compile</phase>
```initialize

Perhaps `pre-compile` here would be better, this would trigger installing node/npm on just running the licence check (I think?)

Do we need `npm` I thought we use only `yarn`, perhaps this whole execution could be removed?

---

My preference here would be to keep the installation to initialize, which is designed to initialize the build state (https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html).